concepticondata/conceptlists/Luangthongkum-2019-341.tsv

```diff
@@ -0,0 +1,342 @@
+Luangthongkum-2019-341-39	39	caress		
+Luangthongkum-2019-341-40	40	carry (on the back)	465	CARRY ON BACK
+Luangthongkum-2019-341-41	41	caterpillar	2247	CATERPILLAR
+Luangthongkum-2019-341-42	42	catfish	3777	CATFISH
+Luangthongkum-2019-341-43	43	carapace, shell	598	SHELL
+Luangthongkum-2019-341-44	44	centipede	1974	CENTIPEDE
+Luangthongkum-2019-341-45	45	chameleon	1555	CHAMELEON
+Luangthongkum-2019-341-46	46	charcoal	1188	CHARCOAL
+Luangthongkum-2019-341-47	47	chicken	1318	CHICKEN
+Luangthongkum-2019-341-48	48	chili pepper, pungent	919	CHILI PEPPER
+Luangthongkum-2019-341-49	49	chin, jaw	1510	CHIN
+Luangthongkum-2019-341-50	50	cicada	2353	CICADA
+Luangthongkum-2019-341-51	51	civet cat	3157	CIVET
+Luangthongkum-2019-341-52	52	classifier (hand of bananas)		
+Luangthongkum-2019-341-53	53	classifier for flat objects		
+Luangthongkum-2019-341-54	54	classifier for people, persons		
+Luangthongkum-2019-341-55	55	classifier for a piece of meat, cake, etc.		
+Luangthongkum-2019-341-56	56	classifier for plants, e.g. bamboo, rice, banana etc. (clumps, clusters)		
+Luangthongkum-2019-341-57	57	classfier for round objects		
+Luangthongkum-2019-341-58	58	classifier for long objects		
+Luangthongkum-2019-341-59	59	classifier for four-legged animals		
+Luangthongkum-2019-341-60	60	close (one’s eyes)	2063	CLOSE (EYES)
+Luangthongkum-2019-341-61	61	comb (v.)	2070	COMB (VERB)
+Luangthongkum-2019-341-62	62	cotton thread	1161	THREAD
+Luangthongkum-2019-341-63	63	cough (v.)	879	COUGH
+Luangthongkum-2019-341-64	64	crab	2348	CRAB
+Luangthongkum-2019-341-65	65	cricket	2354	CRICKET
+Luangthongkum-2019-341-66	66	crocodile	1857	CROCODILE
+Luangthongkum-2019-341-67	67	crop (of fowl), gizzard		
+Luangthongkum-2019-341-68	68	cross (v.)	574	TRANSIT
+Luangthongkum-2019-341-69	69	crossbow	3193	CROSSBOW
+Luangthongkum-2019-341-70	70	crow (v.)	2074	CROW (VERB)
+Luangthongkum-2019-341-71	71	crush (v.)	1520	CRUSH
+Luangthongkum-2019-341-72	72	cucumber	496	CUCUMBER
+Luangthongkum-2019-341-73	73	dark, late evening		
+Luangthongkum-2019-341-74	74	daughter-in-law	1057	DAUGHTER-IN-LAW
+Luangthongkum-2019-341-75	75	day	1225	DAY (NOT NIGHT)
+Luangthongkum-2019-341-76	76	dead,die	2101	BE DEAD OR DIE
+Luangthongkum-2019-341-77	77	deaf	996	DEAF
+Luangthongkum-2019-341-78	78	deep	1593	DEEP
+Luangthongkum-2019-341-79	79	deer (barking~)		
+Luangthongkum-2019-341-80	80	deer (sambha ~)		
+Luangthongkum-2019-341-81	81	delicious	1813	TASTY
+Luangthongkum-2019-341-82	82	descent, go down, set	2014	GO DOWN (DESCEND)
+Luangthongkum-2019-341-83	83	dig	1418	DIG
+Luangthongkum-2019-341-84	84	dip up		
+Luangthongkum-2019-341-85	85	Dipterocarpus tuberculatus Roxb		
+Luangthongkum-2019-341-86	86	disappear, lost		
+Luangthongkum-2019-341-87	87	dog	2009	DOG
+Luangthongkum-2019-341-88	88	do, make	2575	DO OR MAKE
+Luangthongkum-2019-341-89	89	dove	1853	DOVE
+Luangthongkum-2019-341-90	90	dream (v.)	1920	DREAM (SOMETHING)
+Luangthongkum-2019-341-91	91	drink(v.)	1401	DRINK
+Luangthongkum-2019-341-92	92	dry(v.)		
```

Comment:
Yes.

Reply:
This is one of the ambiguous cases I mentioned in the description of the PR. The reason I didn't map it to MAKE DRY is that it can in theory equally mean DRY UP.

Reply:
Thanks, @Kristina-Pianykh ! I'd leave it unmaped then.

Reply:
This is a typical problem in ST languages or SEA languages, that they often have an adjective like "dry" that can also be used as a verb. And there, the meaning is fuzzy. Leaving unmapped is a good strategy for the tome being here.